lua/orgmode/config/defaults.lua

```diff
@@ -35,6 +35,8 @@ local DefaultConfig = {
   org_highlight_latex_and_related = nil,
   org_custom_exports = {},
   org_indent_mode = 'indent',
+  org_time_picker_min_big = 15,
```

Comment:
Let's remove these options and rely only on `org_time_stamp_rounding_minutes`.  Holding j/k updates the time fast enough, so we can stick with only one step.

Regarding the rounding, I would suggest going with additional mappings instead of having it as an option. There is a similar feature to be implemented for regular time up/down (https://github.com/nvim-orgmode/orgmode/issues/396) and Emacs uses a mapping with different prefix.

We can either go with uppercase `J/K` or maybe `<leader>j/k`.

Reply:
I actually really would like to keep it, because I like it in the daily usage. My argument is precision. I often schedule meetings to a quarter of an hour, holding  the 5 min key is not the same. If different preferences exist, I rather would make it optional via configuration than passing on it completely.

Reply:
Ok, lets just create a separate `calendar` config option and nest all these settings inside. I want to separate non Orgmode configuration.

Reply:
I wrapped all calendar related settings into a `calendar` table as you suggested.